### PR TITLE
amass-viz: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/amass-viz.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# amass viz
+
+> Visualize gathered information in a network graph.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-viz-subcommand>.
+
+- Genera una visualización D3.js basada en los datos de la base de datos:
+
+`amass viz -d3 -dir {{ruta/a/directorio_de_base_de_datos}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-viz.md:8]:
```suggestion
- Genere una visualización D3.js basada en datos específicos de la base de datos::

`amass viz -d3 -dir {{ruta/a/directorio_de_base_de_datos}}`
```

Optional fix (error source: original page): unexpected `database` term, `specific` expected.
Note: `Generate a D3.js visualization based on specific database data:` can be used as a translation source.

File: pages.es/common/amass-viz.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Genera un archivo DOT a partir de los datos de la base de datos:
+
+`amass viz -dot -dir {{ruta/a/directorio_de_base_de_datos}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-viz.md:12]:
```suggestion
- Genera un archivo DOT a partir de los datos específicos de la base de datos:

`amass viz -dot -dir {{ruta/a/directorio_de_base_de_datos}}`
```

The same issue as above.

File: pages.es/common/amass-viz.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Genera de un archivo en formato Gephi Graph Exchange XML (GEXF) a partir de los datos de la base de datos:
+
+`amass viz -gexf -dir {{ruta/a/directorio_de_base_de_datos}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-viz.md:16]:
```suggestion
- Genera de un archivo en formato Gephi Graph Exchange XML (GEXF) a partir específicos de los datos de la base de datos:

`amass viz -gexf -dir {{ruta/a/directorio_de_base_de_datos}}`
```

The same issue as above.

File: pages.es/common/amass-viz.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Genera un archivo Graphistry JSON a partir de los datos de la base de datos:
+
+`amass viz -graphistry -dir {{ruta/a/directorio_de_base_de_datos}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-viz.md:20]:
```suggestion
- Genera un archivo Graphistry JSON a partir de los datos específicos de la base de datos:

`amass viz -graphistry -dir {{ruta/a/directorio_de_base_de_datos}}`
```

The same issue as above.

File: pages.es/common/amass-viz.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Genera un archivo CSV Maltego a partir de los datos de la base de datos:
+
+`amass viz -maltego -dir {{ruta/a/directorio_de_base_de_datos}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-viz.md:24]:
```suggestion
- Genera un archivo CSV Maltego a partir de los datos específicos de la base de datos:

`amass viz -maltego -dir {{ruta/a/directorio_de_base_de_datos}}`
```

The same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! 🚀

File: pages.es/common/amass-viz.md (hunk 1 above)
Comment (navarroaxel) [reply pages.es/common/amass-viz.md:8]: The current translation with `Genera` is ok.
the DB is specified, no the data. `basada en datos de una específica base de datos` is the correct semantic. but leave as-is.

Comment (navarroaxel) [inline pages.es/common/amass-viz.md:8]:
```suggestion
`amass viz -d3 -dir {{ruta/al/directorio_de_base_de_datos}}`
```

File: pages.es/common/amass-viz.md (hunk 2 above)
Comment (navarroaxel) [inline pages.es/common/amass-viz.md:12]:
```suggestion
`amass viz -dot -dir {{ruta/al/directorio_de_base_de_datos}}`
```

File: pages.es/common/amass-viz.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Genera una visualización D3.js basada en los datos de la base de datos:
+
+`amass viz -d3 -dir {{ruta/a/directorio_de_base_de_datos}}`
+
+- Genera un archivo DOT a partir de los datos de la base de datos:
+
+`amass viz -dot -dir {{ruta/a/directorio_de_base_de_datos}}`
+
+- Genera de un archivo en formato Gephi Graph Exchange XML (GEXF) a partir de los datos de la base de datos:
```
Comment (navarroaxel) [inline pages.es/common/amass-viz.md:14]:
```suggestion
- Genera un archivo en formato Gephi Graph Exchange XML (GEXF) a partir de los datos de la base de datos:
```

File: pages.es/common/amass-viz.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Genere una visualización D3.js basada en datos específicos de la base de datos::
+
+`amass viz -d3 -dir {{ruta/al/directorio_de_base_de_datos}}`
+
+- Genera un archivo DOT a partir de los datos específicos de la base de datos:
+
+`amass viz -dot -dir {{ruta/al/directorio_de_base_de_datos}}`
+
+- Genera de un archivo en formato Gephi Graph Exchange XML (GEXF) a partir específicos de los datos de la base de datos:
```
Comment (navarroaxel) [inline pages.es/common/amass-viz.md:14]:
```suggestion
- Genera un archivo en formato Gephi Graph Exchange XML (GEXF) a partir específicos de los datos de la base de datos:
```

File: pages.es/common/amass-viz.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`amass viz -gexf -dir {{ruta/a/directorio_de_base_de_datos}}`
```
Comment (navarroaxel) [inline pages.es/common/amass-viz.md:16]:
```suggestion
`amass viz -gexf -dir {{ruta/al/directorio_de_base_de_datos}}`
```

File: pages.es/common/amass-viz.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+- Genera un archivo Graphistry JSON a partir de los datos específicos de la base de datos:
+
+`amass viz -graphistry -dir {{ruta/a/directorio_de_base_de_datos}}`
```
Comment (navarroaxel) [inline pages.es/common/amass-viz.md:20]:
```suggestion
`amass viz -graphistry -dir {{ruta/al/directorio_de_base_de_datos}}`
```

File: pages.es/common/amass-viz.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+- Genera un archivo CSV Maltego a partir de los datos específicos de la base de datos:
+
+`amass viz -maltego -dir {{ruta/a/directorio_de_base_de_datos}}`
```
Comment (navarroaxel) [inline pages.es/common/amass-viz.md:24]:
```suggestion
`amass viz -maltego -dir {{ruta/al/directorio_de_base_de_datos}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**